From: Phil Elwell <phil@raspberrypi.org>
Date: Fri, 24 Jan 2020 09:02:37 +0000
Subject: overlays: Use preferred compatible strings

Make sure all overlays have correct compatible strings before enabling
the automated checking.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/boot/dts/overlays/hifiberry-dacplusadcpro-overlay.dts | 2 +-
 arch/arm/boot/dts/overlays/iqaudio-codec-overlay.dts           | 2 +-
 arch/arm/boot/dts/overlays/pwm-2chan-overlay.dts               | 2 ++
 arch/arm/boot/dts/overlays/pwm-overlay.dts                     | 2 ++
 arch/arm/boot/dts/overlays/smi-dev-overlay.dts                 | 2 ++
 5 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/overlays/hifiberry-dacplusadcpro-overlay.dts b/arch/arm/boot/dts/overlays/hifiberry-dacplusadcpro-overlay.dts
index 00e5d450a88b..07e70ab62cd4 100644
--- a/arch/arm/boot/dts/overlays/hifiberry-dacplusadcpro-overlay.dts
+++ b/arch/arm/boot/dts/overlays/hifiberry-dacplusadcpro-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target-path = "/clocks";
diff --git a/arch/arm/boot/dts/overlays/iqaudio-codec-overlay.dts b/arch/arm/boot/dts/overlays/iqaudio-codec-overlay.dts
index ff3ef3942c6c..9110f5d34298 100644
--- a/arch/arm/boot/dts/overlays/iqaudio-codec-overlay.dts
+++ b/arch/arm/boot/dts/overlays/iqaudio-codec-overlay.dts
@@ -3,7 +3,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&i2s>;
diff --git a/arch/arm/boot/dts/overlays/pwm-2chan-overlay.dts b/arch/arm/boot/dts/overlays/pwm-2chan-overlay.dts
index abdeddd0f2c8..4ddbbfa04065 100644
--- a/arch/arm/boot/dts/overlays/pwm-2chan-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pwm-2chan-overlay.dts
@@ -17,6 +17,8 @@ N.B.:
 */
 
 / {
+	compatible = "brcm,bcm2835";
+
 	fragment@0 {
 		target = <&gpio>;
 		__overlay__ {
diff --git a/arch/arm/boot/dts/overlays/pwm-overlay.dts b/arch/arm/boot/dts/overlays/pwm-overlay.dts
index 27809e8dc746..92876ab3bc8c 100644
--- a/arch/arm/boot/dts/overlays/pwm-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pwm-overlay.dts
@@ -15,6 +15,8 @@ N.B.:
 */
 
 / {
+	compatible = "brcm,bcm2835";
+
 	fragment@0 {
 		target = <&gpio>;
 		__overlay__ {
diff --git a/arch/arm/boot/dts/overlays/smi-dev-overlay.dts b/arch/arm/boot/dts/overlays/smi-dev-overlay.dts
index b610d8283608..bafab6c92506 100644
--- a/arch/arm/boot/dts/overlays/smi-dev-overlay.dts
+++ b/arch/arm/boot/dts/overlays/smi-dev-overlay.dts
@@ -5,6 +5,8 @@
 /plugin/;
 
 /{
+	compatible = "brcm,bcm2835";
+
 	fragment@0 {
 		target = <&soc>;
 		__overlay__ {
